docs/concepts/privacy.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy in IPFS and why it does not come with a built-in privacy layer or encryption.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As a protocol for peer-to-peer data storage and delivery, IPFS is by design a _public network_: Nodes participating in the network store data affiliated with globally consistent [content addresses](/concepts/content-addressing) (CIDs) and advertise that they have those CIDs available for other nodes to use through publicly viewable [distributed hash tables](/concepts/dht/) (DHTs). This paradigm is one of IPFS's core strengths — at its most basic, it's essentially a globally distributed "server" of the network's total available data, referenceable both by the content itself (those CIDs) and by the participants (the nodes) who have or want the content.
```

Comment:
If the text after a colon can stand on its own as a complete sentence, capitalize the first word. If it can't, leave lowercase. So ...

_This much I know to be true: My cat is the softest cat ever._
but
_My cat: so soft._